campfire/src/web/browser.rs

```diff
@@ -74,10 +81,16 @@ fn spawn(spki: &str, _url: &str) -> anyhow::Result<()> {
 }
 
 #[cfg(target_os = "linux")]
-fn spawn(spki: &str, url: &str) -> anyhow::Result<()> {
+fn spawn(spki: &str, url: &str, options: BrowserOptions) -> anyhow::Result<()> {
     const CANDIDATES: &[&str] = &["google-chrome", "google-chrome-stable", "chromium"];
+    let provided_candidate = &[options.app.as_deref().unwrap_or("google-chrome")];
+    let candidates = if options.app.is_some() {
```

Comment:
👍 Good strategy to disable fallbacks if something explicit is passed, and not the other way around.

Have fallen victim of "you think you specified an override, but it was incorrect so a fallback was silently used for 3 months" *many* times 😅